### PR TITLE
Launch week videos

### PR DESCRIPTION
**Pull request by Abhishek-More**
## Summary

Added video links to each launch day

image

## How did you test this change?

Click-testing

## Are there any deployment considerations?

Nope

## Does this work require review from our design team?

No

### CODE REVIEW

File: highlight.io/components/Launch/Week3/DayFive.tsx [hunk 1]
```diff
@@ -23,23 +25,44 @@ const DayFive = () => {
...
+						className="text-darker-copy-on-dark"
+						type="copy3"
+						emphasis
+					>
+						Day 5: October 20th
+					</Typography>
+
+					<a
+						className="flex items-center gap-1 border border-[#34343A] text-darker-copy-on-dark px-4 py-1 rounded-full"
+						href="https://drive.google.com/file/d/1D13VqO4u4QfYq1Lz8xIIpd_OIjAddkYf/view?usp=sharing"
```
Comment (Vadman97) [inline highlight.io/components/Launch/Week3/DayFive.tsx:40]: shouldn't we be linking to the https://www.youtube.com/@highlight-io/videos videos @Abhishek-More ?

**Review by Vadman97 — APPROVED** [review]

File: highlight.io/components/Launch/Week3/DayFive.tsx (hunk 1 above)
Comment (Abhishek-More) [reply highlight.io/components/Launch/Week3/DayFive.tsx:40]: O, I didn't know this existed. I asked Chris about it and he sent me some google drive links. Will change now

Comment (Abhishek-More) [reply highlight.io/components/Launch/Week3/DayFive.tsx]: Changed

Comment (Vadman97) [reply highlight.io/components/Launch/Week3/DayFive.tsx]: ty!

**Pull request merged**